modules/op-installing-tkn-on-linux-using-rpm.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/tkn_cli/installing-tkn.adoc
+
+[id="installing-tkn-on-linux-using-rpm"]
+
+= Installing {pipelines-title} CLI (tkn) on Linux using an RPM
+
+For {op-system-base-full}, you can install the {pipelines-title} CLI (`tkn`) as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+.Prerequisites
+
+* Must have root or sudo privileges.
+
+.Procedure
+
+. Register with Red Hat Subscription Manager:
++
+----
+# subscription-manager register
+----
+
+. Pull the latest subscription data:
++
+----
+# subscription-manager refresh
+----
+
+. List the available subscriptions:
++
+----
+# subscription-manager list --available --matches '*OpenShift*'
```

Comment:
`sudo subscription-manager list --available --matches '*Openshift'` lists many subscriptions, I think `sudo subscription-manager list --available --matches '*pipelines'` would give proper result.
@ppitonak WDYT?

Reply:
IMHO matching both `OpenShift` and `Pipelines` (upper or lower case) should work but I cannot install with most of OpenShift SKUs. @siamaksade